apps/dot_ext/models.py

```diff
@@ -29,8 +29,8 @@ class Application(AbstractApplication):
     updated = models.DateTimeField(auto_now=True)
     op_tos_uri = models.CharField(default=settings.TOS_URI, blank=True, max_length=512)
     op_policy_uri = models.CharField(default="", blank=True, max_length=512)
-    client_uri = models.CharField(default="", blank=True, max_length=512, verbose_name="Client URI",
-                                  help_text="This is typically a homepage for the application.")
+    website_uri = models.URLField(default="", blank=True, max_length=512, verbose_name="Website URI",
```

Comment:
I think this is going to require a migration, OR specify [`db_column`](https://docs.djangoproject.com/en/2.1/ref/models/fields/#db-column) which might let the database stay the same.

Reply:
Forgot to add the migration. I ran and tested it locally.